include/constants.h

```diff
@@ -4,6 +4,7 @@ namespace reinforcement_learning {  namespace name {
       const char *const  APP_ID                  = "appid";
       const char *const  MODEL_SRC               = "model.source";
       const char *const  MODEL_BLOB_URI          = "model.blob.uri";
+      const char* const  MODEL_HTTP_APPI_HOST    = "model.http.api.host";
```

Comment:
MODEL_HTTP_API_HOST?

Reply:
it's when /model api endpoint is used instead of blob Uri